opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -42,4 +42,18 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successful export request.
+  // When the request is only partially successful (due to e.g. invalid/bad data),
+  // the server MUST populate the partial_success property with a
+  // developer-facing error message along with the number of accepted log records.
+  ExportLogsPartialSuccess partial_success = 1;
+}
+
+message ExportLogsPartialSuccess {
```

Comment:
Why another message instead of embedding it in the parent message?

Reply:
You mean why not add the new fields directly on the existing `Export*ServiceResponse` types? I mentioned that in the PR description with more detail, but in a nutshell it's that senders will have to check if the new fields exist before accessing them (for backwards compatibility) as they don't know which proto version the receivers are returning. Having all new fields in a new message, makes the check easier, as in if the message is `null` it means either no problem or the receiver is using an older proto version. Whereas if we have the fields on the parent, existing message, which fields should receivers use to check if a "partial success" was returned?

Reply:
Hi @bogdandrutu does ^ makes sense to you?